src/OrbitVulkanLayer/VulkanLayerController.h

```diff
@@ -30,7 +30,7 @@ namespace orbit_vulkan_layer {
  * and if we do the proper bootstrapping.
  */
 template <class DispatchTable, class QueueManager, class DeviceManager, class TimerQueryPool,
-          class SubmissionTracker>
+          class SubmissionTracker, typename VulkanWrapper>
```

Comment:
```suggestion
          class SubmissionTracker, class VulkanWrapper>
```
Consistency with the other parameters.